### ISSUE TITLE
Crash in AntlrVSIX.File.ViewCreationListener

### ISSUE THREAD

**Issue by jacosoft** [kaby76/AntlrVSIX#26]
Hi kaby76

I get a crash in VS2019 when trying to open a g4 file after AntlrVSIX installed...

I found the below in Windows Logs ... any idea where I can start looking for the problem?

Thank you in advance, I really look forward to start using AntlrVSIX!

Application: devenv.exe Framework Version: v4.0.30319 Description: The process  was terminated due to an unhandled exception. Exception Info:  System.NullReferenceException at  AntlrVSIX.File.ViewCreationListener+<VsTextViewCreated>d__6.MoveNext() at  System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0(System.Object)  at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate,  System.Object, Int32) at  System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object,  System.Delegate, System.Object, Int32, System.Delegate) at  System.Windows.Threading.DispatcherOperation.InvokeImpl() at  System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(System.Object)  at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)  at  System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext,  System.Threading.ContextCallback, System.Object, Boolean) at  System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext,  System.Threading.ContextCallback, System.Object, Boolean) at  System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext,  System.Threading.ContextCallback, System.Object) at  MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext,  System.Threading.ContextCallback, System.Object) at  System.Windows.Threading.DispatcherOperation.Invoke() at  System.Windows.Threading.Dispatcher.ProcessQueue() at  System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr,  Boolean ByRef) at MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr,  Boolean ByRef) at  MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object) at  System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate,  System.Object, Int32) at  System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object,  System.Delegate, System.Object, Int32, System.Delegate) at  System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority,  System.TimeSpan, System.Delegate, System.Object, Int32) at  MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
--

**Comment by kaby76** [comment]
Thanks for reporting the issue. There were two problems with the code. One problem was that the code here should perform a try {} catch {}, then put up a dialog box to request a call stack sent directly to a server I set up for reporting this kind of problem. The second problem is that the way Visual Studio works, project loading and file opening can cause callbacks to occur in any order, and the code has to be robust to handle incomplete information as reported by Visual Studio's api's. Both these problems are now fixed at least in VsTextViewCreated() now. I'll make a release shortly.
